### PR TITLE
fix: correct path for copying js extensions and typings

### PR DESCRIPTION
**Pull request by kellyo-cricut**
tested in CriTrace-bindings and successfully pulled in CriRaster extensions

### CODE REVIEW

File: Sources/FishyJoesExecute/CodeGen.swift [hunk 1]
```diff
@@ -303,7 +303,7 @@ extension CodeGen {
 
                     var tsSources = ["Sources/Generated/NodeInterface/\(config.module).d.ts"]
                     for (moduleName, modulePath) in dependencyPaths {
-                        let path = "\(modulePath)/ts/\(moduleName).extensions.d.ts"
+                        let path = "\(modulePath.dropLast("/Sources".count))/ts/\(moduleName).extensions.d.ts"
```
Comment (cobbal) [inline Sources/FishyJoesExecute/CodeGen.swift]: This seems very fragile. I'd at least feel better if you added `precondition(modulePath.hasSuffix("/Sources"))` to make errors more obvious

**Review by cobbal — CHANGES_REQUESTED** [review]

File: Sources/FishyJoesExecute/CodeGen.swift (hunk 1 above)
Comment (kellyo-cricut) [reply Sources/FishyJoesExecute/CodeGen.swift]: Is that preferable to an `if this ends with "/Sources"` check? [That suffix is added when the dictionary is constructed](https://github.com/cricut/FishyJoes/blob/6785077b65d80dc32771f29742c20c645fee6760/Sources/FishyJoesExecute/CodeGen.swift#L132)

Comment (cobbal) [reply Sources/FishyJoesExecute/CodeGen.swift]: Maybe we should add the "Sources" suffix only when it's needed then

Comment (cobbal) [reply Sources/FishyJoesExecute/CodeGen.swift]: Or you could use "modulePath/../ts"

**Review by cobbal — APPROVED** [review]

**Pull request merged**